spec/index.html

```diff
@@ -7453,14 +7453,18 @@ <h5>langMATCHES</h5>
             <pre class="prototype nohighlight"> <span class="return">xsd:boolean</span>  <span class="operator">langMatches</span> (<span class="type"><span class="type">xsd:string</span></span> <span class="name">language-tag</span>, <span class="type"><span class="type">xsd:string</span></span> <span class="name">language-range</span>)
             </pre>
 
-            <p>Returns `true` if the <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a>
-              (first argument) matches <code>language-range</code> (second argument).
-              Otherwise, the function returns `false`.
-              Matching is performed according to the basic filtering scheme defined in
-              [[RFC4647]] section 3.3.1. <code>language-range</code> is a basic language range per
-              [[[RFC4647]]] [[RFC4647]] section 2.1. A <code>language-range</code> of "*" matches any
-              non-empty <code>language-tag</code> string. Otherwise, the function returns `false`.
+            <p>Returns `true` if the <code>language-tag</code> (a <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a>)
+               matches <code>language-range</code> (a basic language range per [[[RFC4647]]] [[RFC4647]] section 2.1)
+              according to the basic filtering scheme defined in
+              [[RFC4647]] section 3.3.1. Otherwise, the function returns `false`.
+            </p>
+              
+            <p>If either <code>language-tag</code> or <code>language-range</code> are empty
+              (and thus not a valid language tag or language range, respectively),
+              the function returns `false`.
```

Comment:
I would tend to prefer to return an error here. That `LANGMATCHES("", "")` returns `false` seems quite weird to me but returning an error seems more explainable because `""` is not a valid language tag. Also, with an error, the behavior is similar to e.g. `LANGMATCHES(<ex:a>, <ex:b>`

Reply:
As I mentioned in the issue, I don't have strong feelings here, as long as the wording is clear. I think it's easy to see why `""` would result in an error (but have a tougher time with the non-empty error cases, since that might interfere with existing systems and data).

Reply:
> * I'm personally not sure this is an area where we should be encouraging extension.

Observation:
One potential extension would be Extended Language Range.

Let's finish this PR and circle back if that can be shown to be used somewhere.